docs/document/website/versioned_docs/version-1.6.0/components/mobile/mobile-time-picker.md

```diff
@@ -0,0 +1,86 @@
+---
+id: version-1.6.0-mobile-time-picker
+title: MobileTimePicker
+sidebar_label: MobileTimePicker
```

Comment:
Please create the same file with Japanese language at docs/document/website/translated_docs/ja/version-1.6.0